### PR TITLE
#AB34487 Added ScanReportConceptID model

### PR DESCRIPTION
**Pull request by simon-tarr**
This PR adds a new model for NLP conceptIDs

### CODE REVIEW

File: api/mapping/models.py [hunk 1]
```diff
@@ -535,22 +528,40 @@ class DataDictionary(BaseModel):
...
         blank=True,
         null=True,
     )
-    
+
+    def __str__(self):
+        return str(self.id)
+
+
+class ScanReportConceptID(BaseModel):
```
Comment (py5gol) [inline api/mapping/models.py:553]: Can we call it just ScanReportConcept?

File: api/mapping/models.py [hunk 2]
```diff
@@ -535,22 +528,40 @@ class DataDictionary(BaseModel):
...
+    concept_name = models.CharField(max_length=128)
```
Comment (py5gol) [inline api/mapping/models.py:554]: With this, and many other in the models.py, could you break break the model attributes in multiple lines?

**Review by py5gol — CHANGES_REQUESTED** [review]

File: api/mapping/models.py (hunk 2 above)
Comment (simon-tarr) [reply api/mapping/models.py:554]: @py5gol I can do, yes. But my implementation of PEP8 only automatically breaks onto multiple lines when the line length gets to a certain length (not sure about the number of characters though)

Comment (simon-tarr) [reply api/mapping/models.py:554]: I think it also splits onto multiple lines if number of attributes >2

**Review by vpnu — APPROVED** [review]

**Review by py5gol — APPROVED** [review]

**Pull request merged**